Makefile

```diff
@@ -149,7 +149,7 @@ lint-fix: golangci-lint fmt vet ## Run golangci-lint linter and perform fixes
 
 .PHONY: add-license
 add-license: addlicense
-	$(ADDLICENSE) -c "" -ignore ".github/**" -ignore "config/**" -ignore "templates/**" -ignore "bin/**" -ignore ".*" .
+	$(ADDLICENSE) -c "" -ignore ".github/**" -ignore "config/**" -ignore "templates/**" -ignore "providers/**" -ignore "bin/**" -ignore ".*" .
```

Comment:
are we stopping adding copyrights for `providers/` ?

Reply:
Since we don’t set copyrights for the template YAMLs, I thought we could follow the same approach for providers, as those lines seem inaccurate in the resulting ConfigMap. However, this is just my perspective, and I won’t insist on reverting it. Do you think we should keep it?

Reply:
I think that for `licensing` we should get approval, I am OK with removing also

Reply:
Let's keep it as is for now and discuss it with the management team later.